server/src/models/job_posts.model.js

```diff
@@ -0,0 +1,100 @@
+require("dotenv").config();
+const { DataTypes } = require("sequelize");
+const { sequelize } = require("../configs/sequelize");
+const User = require("./user.model");
+
+const JobPosting = sequelize.define(
+  "job_postings",
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true,
+      autoIncrement: true,
+    },
+    title: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    employer: {
+      type: DataTypes.STRING,
+      allowNull: true,
+    },
+    location: {
+      type: DataTypes.STRING,
+      allowNull: true,
+    },
+    hours_per_week: {
+      type: DataTypes.INTEGER,
+      allowNull: true,
+      validate: {
+        min: 1,
+      },
+    },
+    rate_of_pay_min: {
+      type: DataTypes.DECIMAL,
+      allowNull: true,
+    },
+    rate_of_pay_max: {
+      type: DataTypes.DECIMAL,
+      allowNull: true,
+    },
+    rate_of_pay_frequency: {
+      type: DataTypes.ENUM(
+        "hourly",
```

Comment:
To match with the case formatting of the job types, we should capitalize the first letter of each of these options (it will also make rendering the data on the frontend easier)